_content/articles/translation-of-pancagatidipani_hazlewood-a.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Translation of Pancagatidipani"
+authors:
+  - hazlewood-a
+external_url: "https://ia904504.us.archive.org/32/items/jpts-xi-1987/Pancagatidipani_%20Illumination%20of%20the%20Five%20Realms%20of%20Existence%20-%20Ann%20Appleby%20Hazlewood_text.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/10jzF4gZhLTwEwvbB2MHWyI1FHe-9DXDl/view?usp=sharing"
+course: cosmology
+tags:
+  - karma
```

Comment:
let's go for `roots` instead, as this is an excellent example of the Buddhist textual tradition